ci/gpu/build.sh

```diff
@@ -169,7 +169,7 @@ if [[ -z "$PROJECT_FLASH" || "$PROJECT_FLASH" == "0" ]]; then
         done
 
         # Test libcudf (csv, orc, and parquet) with `LIBCUDF_CUFILE_POLICY=KVIKIO`
-        for test_name in "CSV_TEST" "ORC_TEST" "PARQUET_TEST"; do
+        for test_name in "CSV_TEST" "ORC_TEST" "PARQUET_TEST" "DATA_CHUNK_SOURCE_TEST"; do
```

Comment:
This file is no longer used since we've switched to GitHub Actions.

We will be opening PRs to remove all of these dead files from the repository after the `23.02` release.

The file below is where we test the C++ now. These changes should be moved to there

- https://github.com/rapidsai/cudf/blob/branch-23.02/ci/test_cpp.sh

Reply:
Thanks, added it to `test_cpp.sh`. When https://github.com/rapidsai/cudf/pull/12574 get merged, we don't need this anymore.